.github/mergify.yml

```diff
@@ -0,0 +1,80 @@
+---
+defaults:
+  actions:
+    queue:
+      name: default
+      method: rebase
+      update_method: rebase
+
+
+# each test should be listed separately, do not use regular expressions:
+# https://docs.mergify.io/conditions.html#validating-all-status-check
+# TODO: Use mergify's recently added 'shared configuration support'
+# to dedup some of the check-x=y repetition in the future.
+queue_rules:
+  - name: default
+    conditions:
+      - check-success=check
+      - check-success=build
+      - check-success=podmanbuild
+      - check-success=dockerbuild
+      - check-success=test
+      - check-success=test-kubernetes
+      -check-success=centos-ci/sink-clustered/mini-k8s-1.25 
+
+      - check-success=dpulls
+
+
+pull_request_rules:
+  # Clearing approvals after content changes
+  - name: Remove outdated approvals
+    conditions:
+      - base=master
+    actions:
+      dismiss_reviews:
+        approved: true
+        changes_requested: false
+  # Perform automatic merge on conditions
+  - name: Automatic merge on approval
+    conditions:
+      - check-success=check
+      - check-success=build
+      - check-success=podmanbuild
+      - check-success=dockerbuild
+      - check-success=test
+      - check-success=test-kubernetes
+      -check-success=centos-ci/sink-clustered/mini-k8s-1.25 
```

Comment:
not quite sure yet if this will work. it deviates from the pattern of the other checks ...

Reply:
The name appears to match, so I think it'll be all right. If not we can just fix it later.

Reply:
One of the syntax errors is on this line. There should be a space after the first dash.

Reply:
> not quite sure yet if this will work. it deviates from the pattern of the other checks ...

This should work. As John mentioned above a space(after hyphen `-`) is all that is missing from the above line.